src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -102,18 +108,18 @@ export const Disabled: StoryObj<Args> = {
 
 export const WithoutVisibleLabel: StoryObj<Args> = {
   args: {
-    'aria-label': 'a checkbox has no name',
```

Comment:
a [Game of Thrones](https://www.gameofthronesquote.com/arya-stark/a-girl-has-no-name) reference I believe ⚔️ 👑